godtools/Configurations/TractColors.swift

```diff
@@ -8,10 +8,26 @@
 
 import UIKit
 
-class TractColors {
+class TractColors: NSObject {
     
     var primaryColor: UIColor?
     var primaryTextColor: UIColor?
     var textColor: UIColor?
+    
+    override init() {
```

Comment:
I believe so... Normally, I will delete it and see what happens (because I don't remember) but I believe it was needed in order to make the other init (currently I don't have Xcode for testing)